src/BenchmarksServer/Startup.cs

```diff
@@ -1309,22 +1310,27 @@ private static string GetDotNetExecutable(string dotnetHome)
 
         private static Process StartProcess(string hostname, string benchmarksRepo, ServerJob job, string dotnetHome, bool perfview, StringBuilder standardOutput)
         {
+            job.BasePath = Path.Combine(benchmarksRepo, Path.GetDirectoryName(job.Source.Project));
+
             var serverUrl = $"{job.Scheme.ToString().ToLowerInvariant()}://{hostname}:{job.Port}";
-            var dotnetFilename = GetDotNetExecutable(dotnetHome);
+            var executable = GetDotNetExecutable(dotnetHome);
             var projectFilename = Path.GetFileNameWithoutExtension(job.Source.Project);
-            var benchmarksDll = job.UseRuntimeStore ? $"bin/Release/netcoreapp2.0/{projectFilename}.dll" : $"published/{projectFilename}.dll";
+            var benchmarksBin = job.UseRuntimeStore ? $"bin/Release/netcoreapp2.0" : $"published";
```

Comment:
Why is this netcoreapp2.0?

Reply:
No idea.
/cc @sebastienros .

Reply:
The idea is that if we don't do a `dotnet publish` we'll use the runtime store. This was done initially to measure the impact with or without it, but in practice we only measure a published application. We can ignore for now and I'll remove this code path once we decide to get rid of it.

Reply:
@sebastienros: I think @davidfowl is asking why it's `netcoreapp2.1` rather than `netcoreapp2.0`.

Reply:
Now that you say it it's obvious ;) That's an issue then, filing a bug.

Reply:
It would be good to compare the perf of 3 configs:

1. netcoreapp2.0 with 2.0 packages
2. netcoreapp2.0 with 2.1 packages
3. netcoreapp2.1 with 2.1 packages

I'm not sure if there will be any difference between 2 and 3, but 1 and 3 are the recommended customer configurations.

Reply:
Will changing this change the default runtime to 2.1 when I use the driver? If so, 🎉